src/renderer/components/GameHost/TwoOptionsQuestionHost/TwoOptionsQuestionHost.tsx

```diff
@@ -28,6 +28,10 @@ const TwoOptionsQuestionHost: React.FC<GuessYearQuestionHostProps> = ({ question
 
   const [actionButtonType, setActionButtonType] = useState<ActionButtonType>(ActionButtonType.ShowQuestion);
 
+  useEffect(() => {
+    setActionButtonType(ActionButtonType.ShowQuestion);
```

Comment:
why this needs to be used inside useEffect? actionbutton type can change during the life of component, and when question changes, you want to reset to ShowQuestion type?